tests/server/login-test.js

```diff
@@ -0,0 +1,29 @@
+const app = require('../../tools/app.js')
+const request = require('supertest')
```

Comment:
@Bronson-Brown-deVost : If you like this approach, check out the docs for `supertest`.